src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -127,6 +127,10 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
     [visibleTypesInList, selectableTypes],
   )
 
+  // For dual-pane, this state variable indicates which container is selected in the tree view and should be shown in the table view
+  const [currentContainer, setCurrentContainer] =
+    useState<EntityTreeContainer>(initialContainer)
+
```

Comment:
State pulled up from `EntityTree` so that the setter can be shared with the table component